core/tests/test_views.py

```diff
@@ -170,6 +169,13 @@ def test_report_post(client: Client, user: User, post: Post, report_data: dict)
     assert str(messages[0]) == "Your post has been reported."
 
 
+def test_report_post_invalid_data(client: Client, user: User, post: Post) -> None:
+    client.force_login(user)
+    invalid_data = {}
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=invalid_data)
+    assert response.status_code == 400
```

Comment:
This is wrong. Since it is browser it should only check for response.context.form.errors (I am guessing here) not for response code. Also, response code only shows that there is an error, not the details of an error. It would be good to add check for at least error amount.